docs/utils.md

```diff
@@ -0,0 +1,17 @@
+# Utils
```

Comment:
I would either not abbreviate it, use the package name, or both: Utilities `<extra.utils>`

Looking forward, we probably want to include both the human-readable text as well as the package name here. But if you think both is too much, I would probably go with the non-abbreviated human-readable name.

Reply:
Renamed to `Utilities` in 9786f2b.